src/Tizen.NUI/src/public/BaseComponents/FlexContainer.cs

```diff
@@ -27,9 +27,9 @@ namespace Tizen.NUI.BaseComponents
     /// FlexContainer can expand items to fill available free space, or shrink them to prevent overflow.<br />
     /// </summary>
     /// <since_tizen> 3 </since_tizen>
+    [Obsolete("Deprecated in API8, Will be removed in API11, " + "Please use FlexLayout instead!")]
```

Comment:
This second comma is unnecessary. Additionally, I don't see any reason why it couldn't be written as a single line of text.
```suggestion
    [Obsolete("Deprecated in API8, Will be removed in API11. Please use FlexLayout instead!")]
```

Reply:
Removed the comma and changed as a single line.